drivers/linux/eth/ionic/Makefile

```diff
@@ -4,7 +4,7 @@
 obj-$(CONFIG_IONIC) := ionic.o
 obj-$(CONFIG_IONIC_MNIC) := ionic_mnic.o
 
-ccflags-y := -g -I$(M)/../common
```

Comment:
In the linux directory the "make" command fails with this change.

Reply:
Looks like I had stray copies of ionic_if.h and ionic_regs.h in my build directory that allowed my testing to succeed.  I've fixed this in my process steps to be sure they don't get in the way in the future.